torch/csrc/jit/script/module.h

```diff
@@ -155,7 +155,12 @@ struct Method {
     return *this;
   }
 
-  const FunctionSchema& getSchema() const;
+  const FunctionSchema& getSchema() const {
```

Comment:
This is failing because the build is broken, see: https://github.com/pytorch/pytorch/pull/12333/files. Module.cpp is included in both libtorch.so and _C.so.  I don't want to move more things into header files to fix a broken build.